### ISSUE TITLE
Use Authors@R?

### ISSUE THREAD

**Issue by maelle** [igraph/rigraph#609]
In the DESCRIPTION file instead of having Author and Maintainer you could use `Authors@R`. It would allow you to list all authors. pkgdown would pick it up (whereas at the moment it only picks up the maintainer).

Cc @krlmlr

**Comment by maelle** [comment]
Example of a package with many people listed with different roles https://github.com/rstudio/rmarkdown/blob/main/DESCRIPTION

In the pkgdown website you can filter authors shown in the sidebar and in the footer based on roles. In all cases all authors are shown on the author page. https://pkgdown.r-lib.org/reference/build_home.html#authors

**Comment by ntamas** [comment]
I'm totally fine with converting the DESCRIPTION file to `Authors@R`. However -- is it customary to include all the authors in `Authors@R` who has ever sent a pull request to the repository that eventually got merged? If so, is there a tool to automatically update `Authors@R` in such cases? (Something like https://allcontributors.org/ , which we use in the C core and the Python interface already).

**Comment by maelle** [comment]
No, more customary would be to list contributors in the changelog (NEWS.md) instead. {fledge} would help with this.

You could use allcontributors for the README thanks to @mpadge https://github.com/ropenscilabs/allcontributors

**Comment by maelle** [comment]
And so in DESCRIPTION you'd keep only "main" authors.

**Comment by ntamas** [comment]
Sounds fine to me then!
